### ISSUE TITLE
getImagePixels case GC problem.

### ISSUE THREAD

**Issue by finanshow** [mlapadula/GifEncoder#1]
The bigmap size: width: 1280, height:720
When call getImagePixels(), it seems too big to getPixcels and it stop there case Android GC.
 int[] pixelInts = new int[w*h*3];
 image.getPixels(pixelInts, 0,width, 0, 0, w, h);

**Comment by vince-styling** [comment]
yes, for me, it also cause oom if you got too many frames, don't know if there is a way to work around
